src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -41,9 +42,8 @@ public class TypicalPersons {
     public static final Person FIONA = new PersonBuilder().withName("Fiona Kunz").withPhone("9482427")
             .withEmail("lydia@example.com").withAddress("little tokyo")
             .withAdditionalNotes("Remind student to submit homework").build();
-    public static final Person GEORGE = new PersonBuilder().withName("George Best").withPhone("9482442")
-            .withEmail("anna@example.com").withAddress("4th street")
-            .withAdditionalNotes("Remind student to submit homework").build();
+
+    public static final Person GEORGE = constructGeorge();
```

Comment:
Could you explain a bit on why you have abstracted out the constructor into a separate method?

Reply:
This issue has been addressed during private messages. Below explanation is for record:

A separate constructor is used in order to catch the exception that ParseException for the `class` field.

But given that George is only used to test the `addCommand`, the `class` field should not exist in the first place and hence the part has been reverted back to previous version.